### PR TITLE
Added and changed german localization

### PR DESCRIPTION
**Pull request by fridomac**
Added some terms for the new modules and changed "Bericht" to "-Report" (sounds less stilted)

### CODE REVIEW

File: assets/locales/de.json [hunk 1]
```diff
@@ -644,46 +718,121 @@
...
+      },
       "missing": "Fehlt",
       "normal": "Normal",
       "now": "Jetzt austauschen",
       "service": "Jetzt zur Wartung",
       "soon": "Wartung steht an",
-      "nobattery": "Keine Batterie"
+      "nobattery": "Keine Batterie",
+      "title": "Batterie-Zustand",
+      "tooltip": "LIste des Batteriezustands"
```
Comment (bochoven) [inline assets/locales/de.json]: Little typo: LIste

**Comment by bochoven** [comment]
Vielen dank!

**Comment by fridomac** [comment]
Ok, fixed it. Do I have to make a new pull request?

Bye, Frido.

2017-02-14 10:11 GMT+01:00 Arjen van Bochoven <notifications@github.com>:

File: assets/locales/de.json [hunk 2]
```diff
@@ -536,15 +598,26 @@
     "enrollment_status": "Enrollment Status"
   },
   "security": {
-    "security": "Security"
+    "report": "Sicherheits-Report",
+    "security": "Security",
+    "gatekeeper": "",
```
Comment (bochoven) [inline assets/locales/de.json]: Sorry, I did not notice before, but there are empty strings here

Comment (fridomac) [inline assets/locales/de.json]: Ok, on it. I thought that only strings that have a translated word had to be filled in, sorry! Will send new PR.

**Pull request merged**